web/src/lib/components/album-page/user-selection-modal.svelte

```diff
@@ -24,9 +24,9 @@
   let selectedUsers: Record<string, { user: UserResponseDto; role: AlbumUserRole }> = {};
 
   const roleOptions: Array<{ title: string; value: AlbumUserRole | 'none'; icon?: string }> = [
-    { title: $t('editor'), value: AlbumUserRole.Editor, icon: mdiPencil },
-    { title: $t('viewer'), value: AlbumUserRole.Viewer, icon: mdiEye },
-    { title: $t('remove'), value: 'none' },
+    { title: $t('role_editor'), value: AlbumUserRole.Editor, icon: mdiPencil },
+    { title: $t('role_viewer'), value: AlbumUserRole.Viewer, icon: mdiEye },
+    { title: $t('remove_user'), value: 'none' },
```

Comment:
Why not reuse the `remove` key?

Reply:
"Remove a user" sounds odd in other languages(e.g. cz).

Reply:
Ah that makes sense, we could also make it clearer by changing it to "Remove user"